### PR TITLE
Fix Code Climate issues with `Conversation View`

### PR DESCRIPTION
**Pull request by nisusam**
- Fix https://github.com/chatwoot/chatwoot/issues/66
- Fix https://github.com/chatwoot/chatwoot/issues/68
- Fix https://github.com/chatwoot/chatwoot/issues/69

### CODE REVIEW

File: app/javascript/src/assets/scss/widgets/_conversation-view.scss [hunk 1]
```diff
@@ -2,7 +2,7 @@
   @include flex;
   @include flex-direction(column);
 
-  .chat-list__top {
+  .chat-list-top {
```
Comment (pranavrajs) [inline app/javascript/src/assets/scss/widgets/_conversation-view.scss]: @nisusam We follow BEM for CSS, could you please update the classes to comply with that?
http://getbem.com/introduction/

Comment (nisusam) [reply app/javascript/src/assets/scss/widgets/_conversation-view.scss]: @pranavrajs Updated this PR

File: app/javascript/src/assets/scss/widgets/_conversation-view.scss [hunk 2]
```diff
@@ -119,14 +122,15 @@
...
-        color: $color-heading;
-        margin-right: auto;
         border-bottom-left-radius: 0;
         border-top-left-radius: 0;
+        color: $color-heading;
+        margin-right: auto;
       }
 
-      &+.right {
+      +.right {
```
Comment (nithindavid) [inline app/javascript/src/assets/scss/widgets/_conversation-view.scss]: @nisusam `&` is omitted here. Is there any reason for it ? Thanks

File: app/javascript/src/assets/scss/widgets/_conversation-view.scss [hunk 3]
```diff
@@ -138,32 +142,34 @@
...
             color: $medium-gray;
+            position: absolute;
+            right: $space-one;
+            top: $space-smaller + $space-micro;
           }
         }
       }
 
-      &+.left {
+      +.left {
```
Comment (nithindavid) [inline app/javascript/src/assets/scss/widgets/_conversation-view.scss]: @nisusam Here also

File: app/javascript/src/assets/scss/widgets/_conversation-view.scss (hunk 2 above)
Comment (nisusam) [reply app/javascript/src/assets/scss/widgets/_conversation-view.scss]: @nithindavid The following error I was getting after running `scss-lint`

```
UnnecessaryParentReference: Unnecessary parent selector (&)
```

Comment (nithindavid) [reply app/javascript/src/assets/scss/widgets/_conversation-view.scss]: Looks like `&` is not needed. Thanks :)

**Comment by nisusam** [comment]
> We're sending stickers to our contributors :D Drop us your info to get in touch with you. Thanks for the PR 🎉

@nithindavid How I can share my info?

**Comment by pranavrajs** [comment]
@nisusam Please share it here https://chatwoot.typeform.com/to/ihG9Gz

**Pull request merged**